atef/widgets/config/utils.py

```diff
@@ -686,3 +799,67 @@ def cast_dataclass(data: Any, new_type: Type) -> Any:
         if key in field_names
     }
     return new_type(**new_kwargs)
+
+
+class MultiInputDialog(QtWidgets.QDialog):
```

Comment:
Missing brief docstring, what is this for?

Reply:
I'll add that, this is for the input widget that pops up after setting the report file name, lets the user set the header/footer/version/author info.